DataImport.Web/Features/ApiServers/RefreshDataModel.cs

```diff
@@ -0,0 +1,80 @@
+using DataImport.Models;
+using DataImport.Web.Helpers;
+using DataImport.Web.Services;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DataImport.Web.Features.ApiServers
+{
+    public class RefreshDataModel
+    {
+        public class Command : IRequest<Response>
+        {
+            public int ApiServerId { get; set; }
+        }
+
+        public class Response : ToastResponse
+        {
+            public int ApiServerId { get; set; }
+        }
+
+        public class CommandHandler : IRequestHandler<Command, Response>
+        {
+            private readonly DataImportDbContext _database;
+            private readonly IConfigurationService _configurationService;
+
+            public CommandHandler(DataImportDbContext database, IConfigurationService configurationService)
+            {
+                _database = database;
+                _configurationService = configurationService;
+            }
+
+            public async Task<Response> Handle(Command request, CancellationToken cancellationToken)
+            {
+                var apiServer = await _database.ApiServers
+                    .Include(x => x.ApiVersion)
+                    .SingleOrDefaultAsync(x => x.Id == request.ApiServerId, cancellationToken);
+
+                if (apiServer is null)
+                {
+                    throw new System.ArgumentException("Api server not found");
+                }
+
+                await _configurationService.FillSwaggerMetadata(apiServer);
+
+                await _database.SaveChangesAsync(cancellationToken);
+
+                var dataMaps = await (from dm in _database.DataMaps
```

Comment:
@beto-devnet Please update to lambda syntax

Reply:
updated to labdasyntax